flexget/plugins/api/user.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import unicode_literals, division, absolute_import
+
+from flask import request
+from flask.ext.login import current_user
+
+from flexget.api import api, APIResource
+from flexget.webserver import change_password, generate_token, WeakPassword
+
+user_api = api.namespace('user', description='Manage user login credentials')
+
+user_password_input = {
+    'type': 'object',
+    'properties': {
+        'password': {'type': 'string'}
+    }
+}
+user_password_input_schema = api.schema('user_password_input', user_password_input)
+
+
+@user_api.route('/')
+@api.doc('Change user password')
+class UserManagementAPI(APIResource):
+    @api.validate(model=user_password_input_schema, description='Password change schema')
+    @api.response(400, 'Password not strong enough')
+    def put(self, session=None):
+        """ Change user password """
+        user = current_user
```

Comment:
being picky here but why copy current_user to user? Why not just use current_user directly?

Also, maybe we should check to ensure current_user is not None? In case the login session times out during the request or something